server/controllers/entities/lib/properties/properties_config_bases.coffee

```diff
@@ -25,6 +25,7 @@ module.exports =
   workEntity: restrictedEntityType 'work'
   serieEntity: restrictedEntityType 'serie'
   humanEntity: restrictedEntityType 'human'
+  countryEntity: restrictedEntityType 'countrie'
```

Comment:
what about using `country` as the singular form? I think the added cost is acceptable, see proposed implementation d5abf5a

Reply:
it hurts my eyes less to see `countrie` here, than a `if pluralType is 'countries' then 'country'` 
its a hack but at least it stays with the other hardcoded types above